app/Config.hs

```diff
@@ -44,8 +43,13 @@ data CopyThing = CopyThing FilePath FilePath
 instance FromJSON Config where
   parseJSON = withObject "Hapistrano configuration" $ \o -> do
     configDeployPath <- o .: "deploy_path"
-    configHost       <- o .:? "host"
-    configPort       <- o .:? "port" .!= 22
+    host             <- o .:? "host"
+    port             <- o .:? "port" .!= 22
+    hs               <- (o .:? "targets" .!= []) >>= mapM (\m -> do
+      host' <- m .: "host"
+      port' <- m .:? "port" .!= 22
```

Comment:
@mrkkrp in order to keep consistency between lines 46-47 and lines 49-50. Maybe we could extract out some of the code to separate functions an reuse it consistently on both cases? That way if we decided to update the default values or something like that we do the change on a single place.